examples/per_file/bin/decrypt-config.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Exit on first error
+set -e
+
+# Define our secret extenssion
+secret_ext=".json"
+
+# If there is a config directory, then move it to a backup
+if test -d config; then
+  if test -d config.bak; then
+    rm -r config.bak
+  fi
+  mv config/ config.bak/
+fi
+
+# Create our new config directory
+mkdir config
+
+# For each of our files in our encrypted config
+for file in $(ls config.enc); do
```

Comment:
I recently found a better way to do this `for file in $(ls` that doesn't break when a file has a space in it. Updating the PR